server/utils/applications/withdrawables/index.test.ts

```diff
@@ -70,14 +70,18 @@ describe('withdrawableTypeRadioOptions', () => {
       ).toEqual([
         {
           text: paWithdrawable.dates
-            .map(datePeriod => DateFormats.formatDurationBetweenTwoDates(datePeriod.startDate, datePeriod.endDate))
+            .map(datePeriod =>
+              DateFormats.formatDurationBetweenTwoDates(datePeriod.startDate, datePeriod.endDate, { format: 'short' }),
```

Comment:
It looks like this is now showing dates in the format like:

> 23/02/2024 - 26/02/2024

Might it be significantly easier for users if we retained the day of the week in our short format e.g.

> Fri 23 Feb 2024 - Mon 26 Feb 2024